services/src/messaging/src/main/java/org/openkilda/messaging/info/flow/FlowRerouteResponse.java

```diff
@@ -15,12 +15,31 @@
 
 package org.openkilda.messaging.info.flow;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonProperty;
 import org.openkilda.messaging.Utils;
+import org.openkilda.messaging.info.InfoData;
 import org.openkilda.messaging.info.event.PathInfoData;
 
-public class FlowRerouteResponse extends FlowPathResponse {
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.Value;
+
+/**
+ * Represents a flow reroute northbound response.
+ */
+@JsonSerialize
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Value
+public class FlowRerouteResponse extends InfoData {
+
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * The response payload.
+     */
+    @JsonProperty(Utils.PAYLOAD)
+    protected PathInfoData payload;
```

Comment:
I thought we have confirmed usage of `BidirectionalFlow` here. Or it will be part of next patch?

Reply:
Will be done in https://github.com/telstra/open-kilda/issues/1023